magento/adobe-stock-integration
# #116 : Fixed issue for Configuration can be saved without valid API key

### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#116: Configuration can be saved with not a valid API key

### Manual testing scenarios (*)

1. Execute magentoCLI command
2. php bin/magento config:set adobe_stock/integration/api_key ""
3. Execute magentoCLI command
4. php bin/magento config:set adobe_stock/integration/enabled 0
5. Before - LoginAsAdmin
6. User navigates to Stores > Settings > Configuration
7. Page reloads
8. User expands Advanced tab
9. User clicks on System section
10. Page reloads
11. Location is admin/system_config/edit/section/system/
12. User expands "Adobe Stock Integration" expandable section
13. User sets "Enabled Adobe Stock" to "Yes"
14. Find input field labeled by Api Key label
15. Input field is in empty state
16. User fills next string to the API Key input adobeadobeadobeadobeadobeadobe
17. User clicks Test Connection button Key is invalid. Try a different key. appears
18. User clicks Connection failed! Test again? button Key is invalid. Try a different key. appears
19. User clicks Save Config button
20. User gets the error message "Key is invalid. Try a different key" and value is not saved
